README.md

```diff
@@ -24,8 +24,8 @@ These preview libraries can be easily identified by their folder, package, and n
 The libraries released in the July 2019 preview:
 * [Azure.ApplicationModel.Configuration](/sdk/appconfiguration/Azure.ApplicationModel.Configuration/README.md)
 * [Azure.Messaging.EventHubs](/sdk/eventhub/Azure.Messaging.EventHubs/README.md)
-* [Azure.Identity.KeyVault.Keys](/sdk/keyvault/Azure.Security.KeyVault.Keys/Readme.md)
-* [Azure.Identity.KeyVault.Secrets](/sdk/keyvault/Azure.Security.KeyVault.Secrets/Readme.md)
+* [Azure.Security.KeyVault.Keys](/sdk/keyvault/Azure.Security.KeyVault.Keys/README.md)
```

Comment:
This rename is likely go cause some grief on windows. What prompted this update?

Reply:
It seems to be a typo in the first place. I asked @schaabs and he said it must have been a typo as there is no package named Azure.Identity.*

Reply:
Updating from Readme.md to README.md is so that the PackageProj changes can consistently apply to all the package readme's. i.e. they all need to use the same case for the link to work.